watchlist.json

```
@@ -1 +1 @@
-{"watchlists": []}
\ No newline at end of file
+{"watchlists": [{"name": "t", "media": {"q": {"status": "unwatched", "tags": []}}, "participants": []}]}
```

Reset the watchlist.json just so it starts from a fresh state each time :)

```suggestion
{"watchlists": []}
```